.github/workflows/release-manual.yaml

```
@@ -24,6 +24,9 @@ jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
```

https://shopify.slack.com/archives/C089605MPL7/p1738076946528959

```suggestion
      id-token: write
      contents: write
      pull-requests: write
```